roles/ngi_pipeline/templates/create_ngi_pipeline_dirs.sh.j2

```diff
@@ -17,7 +17,7 @@ fi
 mkdir -p {{ proj_root }}/$1/private/ngi_pipeline/log
 mkdir -p {{ proj_root }}/$1/private/ngi_pipeline/db
```

Comment:
I could have sworn I edited those two lines, that was nonetheless the intent.
I saw no conflicts with doing such, and see no conflicts with what you grepped either.
I also changed the log name of ngi_reports. I guess it caused no conflict, but it was counter-intuitive.